scipy/spatial/transform/rotation.py

```diff
@@ -172,6 +174,59 @@ def _elementary_quat_compose(seq, angles, intrinsic=False):
     return result
 
 
+def _icosahedral():
```

Comment:
You can store this code in a separate file.

Reply:
Done.